Source/Locales/Menu/es.po

```diff
@@ -658,67 +657,67 @@ msgid "OK"
 msgstr "Aceptar"
 
 #: ../../Menu/Options.Designer.cs:308
-msgid "Brake pipe charging rate (PSI/s)             "
```

Comment:
Will the translation work if the original strings are modified like this? There are a few of this in your changes.